src/store/toast-store.js

```diff
@@ -57,4 +57,9 @@ store.registerHandler('CHECKOUT_SUCCESS', () => {
     store.emitChange();
 });
 
+store.registerHandler('ITEM_DELETED', data => {
+    addToast(`Item ${data.itemAddress} was deleted successfully!`);
```

Comment:
>I don't know what item iGwEZUvfA is. Why can't I just get a model name??
>—User

Add (or replace the address) the model name of the item that was destroyed. (see https://github.com/TheFourFifths/consus-client/pull/42#pullrequestreview-11693081)